Contributor docs: loosen restrictions on caniuse correspondence (#1915)
web-platform-dx/web-features: docs/guidelines.md

- Do not set a `caniuse` value if the Can I Use feature is merely related to the feature.
  For example, in [`grid.yml`](../features/grid.yml), do not set `caniuse: css-subgrid`.

> - Do not set a `caniuse` value if the top-level headline `status` would not accurately reflect whether each browser is shown as supported on Can I Use.

Author: OK, I see what you mean. What I'm trying to get across is that you can't have a browser shown as unsupported on caniuse but supported in web-features (or the opposite). What do you think of something like this instead?

```suggestion
- Do not set a `caniuse` value if the top-level headline `status` does not match Can I Use on whether each browser supports or does not support the feature.
```

> - Do not set a `caniuse` value if the top-level headline `status` would not accurately reflect whether each browser is shown as supported on Can I Use.

Reviewer: Seems good to me!

>   For example, if Can I Use shows that a core browser set browser does not support a feature but web-features's status reports that the feature is Baseline high, then either do not set the `caniuse` value for that feature or submit a correction to Can I Use or browser-compat-data.

Reviewer: ```suggestion
  For example, if Can I Use shows that a browser from the core browser browser set does not support a feature but web-features's status reports that the feature is Baseline high, then:
  * Either do not set the `caniuse` value for that feature.
  * Or submit a correction to the Can I Use data.
  * Or submit a correction to the browser-compat-data that web-features uses.
```

>   For example, if Can I Use shows that a core browser set browser does not support a feature but web-features's status reports that the feature is Baseline high, then either do not set the `caniuse` value for that feature or submit a correction to Can I Use or browser-compat-data.

Author: Good suggestions. I've made some small tweaks and committed this as ed46c1f7.
